src/libs/H.Ipc.Generator/H.Ipc.Generator.csproj

```diff
@@ -12,7 +13,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.3.1" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" PrivateAssets="all" />
```

Comment:
What is the minimum version we need?
The problem here is that updating this package significantly narrows the list of supported IDEs. The most optimal at the moment is 4.3.1, if we do not need any of the versions above

Reply:
I will have another look at this to see if I can set it back to 4.3.1. The updated version is most likely a result of trying to fix some package version inconsistency among packages/projects.